client/src/features/Cart/pages/CartPage/CartPage.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import { Box, Container, Stack, styled, Typography } from '@mui/material';
+import { CartItem } from '../../components';
+import TotalInfoDialog from '../../components/TotalInfoDialog/TotalInfoDialog';
```

Comment:
why didnt you export this component from the index.js file created inside the components folder of this feature?

Reply:
make sense) My fault. Fixed